src/engine/bank01.asm

```diff
@@ -162,7 +162,7 @@ MainDuelLoop:
 
 .not_link_duel
 	ld b, DUEL_ANIM_DUEL_WIN
-	ld c, MUSIC_MATCH_VICTORY
```

Comment:
This change will break the build since you didn't change in the constants file

Reply:
Never mind, this was probably because of a mistake in my part

Reply:
Yeah this was a build breaker from the current repo state.